examples/exit-handler-teams.yaml

```diff
@@ -0,0 +1,33 @@
+# This example shows how to hit a teams webhook on an exit handler.
```

Comment:
Suggested rephrase: "This example shows how to send a Microsoft Teams notification with an exit handler by hitting a Teams webhook." 

also, "... create a Microsoft Teams webhook url for it."

Reply:
@caelan-io done

Reply:
awesome, thanks!